src/redux/redux-store.js

```diff
@@ -1,7 +1,17 @@
-import { combineReducers, createStore } from 'redux'
+import { createStore, applyMiddleware } from 'redux'
+import createSagaMiddleware from 'redux-saga'
+import { composeWithDevTools } from 'redux-devtools-extension'
+import watchFetchingSaga from './saga'
+import rootReducers from './reducers'
 
-const reducers = combineReducers({})
-
-const store = createStore(reducers)
+const initialState = {}
+const sagaMiddleware = createSagaMiddleware()
+const middleware = [sagaMiddleware]
 
+const store = createStore(
+  rootReducers,
+  initialState,
+  composeWithDevTools(applyMiddleware(...middleware)),
+)
+sagaMiddleware.run(watchFetchingSaga)
 export default store
```

Comment:
add some empty lines between these, its hard to read:
```
const store = createStore(
  rootReducers,
  initialState,
  composeWithDevTools(applyMiddleware(...middleware)),
)

sagaMiddleware.run(watchFetchingSaga)

export default store
```

Reply:
Ok. I have corrected